src/fsharp/FSharp.Compiler.Scripting/TextHelpers.fs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.Scripting
+
+open System.Collections.Generic
+open System.IO
+open System.Text
+
+type internal CapturedTextReader() =
```

Comment:
How do these relate to https://github.com/dotnet/fsharp/blob/14be76788d84639b8ed11bfcaf3c36d741e4902c/src/fsharp/fsi/fsi.fs#L2940?

I'm fine with throwing away the old implementations or duplication - just wondering

Reply:
The answer that I should give:
I didn't want to strip `CompilerInputStream` out of `fsi.fs` and put it somewhere that this new assembly could consume it, especially since it's really just an internal implementation detail.

The real answer:
I never even looked so I didn't know it existed.

I don't care too much either way; leaving as is, or moving that type to compiler.private.